pkgs/tools/networking/strongswan/default.nix

```diff
@@ -53,6 +54,9 @@ stdenv.mkDerivation rec {
     substituteInPlace src/swanctl/swanctl.h --replace "SWANCTLDIR" "\"/etc/swanctl\""
     # glibc-2.26 reorganized internal includes
     sed '1i#include <stdint.h>' -i src/libstrongswan/utils/utils/memory.h
+
+    substituteInPlace src/starter/netkey.c --replace 'modprobe ' '${kmod}/bin/modprobe '
+    substituteInPlace src/starter/klips.c --replace 'modprobe ' '${kmod}/bin/modprobe '
```

Comment:
Perhaps add `"` to the pattern so won't accidentally replace code which matches `modprobe `, eg. something like this:   `'"modprobe ' '"${kmod}/bin/modprobe '`.

ref: [starter/netkey.c](https://git.strongswan.org/?p=strongswan.git;a=blob;f=src/starter/netkey.c;h=b150d3e80beed726c944c1779268e847654c16fd;hb=HEAD)

Reply:
I think using a patch with `substituteAll` would be even nicer.

Reply:
@hedning Thanks, that's actually what I originally had, but I changed to the current version to make it a little easier to understand.

@jtojnar Is it general nixpkgs policy to prefer patches over `substituteInPlace`?  As a contributor it is definitely easier to just hack something together with `substituteInPlace`, but maybe actual patches are the recommended way to do something like this?

Reply:
I do not think there is any official policy but many maintainers prefer patches since it makes it easier to notice when the source changes.

Reply:
@jtojnar I've made the change to use `substituteAll` with an actual `.patch` file instead of `substituteInPlace` in commit 2449f5c.

I think this should be ready for one more final review and then merging.